src/commands/bot/createemoji.js

```diff
@@ -5,7 +5,7 @@ module.exports = class CreateEmoji extends Command {
     super({
       name: 'createemoji',
       aliases: ['newemoji'],
-      category: 'utility',
+      category: 'bot',
```

Comment:
I do not agree with this change (along with others like this).
The bot category should only contain commands that are actually related with the bot itself, such as ping, stats, help, commandsource, etc

Reply:
Fair point, I'll revert those changes.